contrib/completion/bash/docker

```diff
@@ -739,6 +739,7 @@ _docker_daemon() {
 		--ip-masq=false
 		--iptables=false
 		--ipv6
+		--raw-logs=false
```

Comment:
Please change to `--raw-logs`. The `=false ` variant is only used for inverse options (defaulting to `true`).